lib/rubygems/commands/query_command.rb

```diff
@@ -17,9 +22,8 @@ def deprecation_warning
     alert_warning message unless Gem::Deprecate.skip
   end
 
-  def initialize(name = 'query',
-                 summary = 'Query gem information in local or remote repositories')
-    super name, summary,
+  def initialize
+    super NAME, SUMMARY,
```

Comment:
These changes are not actually used in the end, right? Let's revert them to keep the structure of command files consistent.

Reply:
It's kind of a mix, TBH. `QueryCommand` is the only command that has arguments given to `#initialize`.  The rest of the commands have a structure similar to:
```ruby
  def initialize
    super 'open', 'Open gem sources in editor'
```

Unless I hear otherwise from you, I'll conform to the style of the other commands. This would result in this chunk of code being changed, but it would increase consistency, which seems to be a priority here 😄 If you prefer that they be fully reverted, I'll do that 👍

Reply:
Normally my preference is to introduce the minimal set of changes to fix the issue, and do any refactorings or consistency improvements separately. Separated commits are fine though if the changes are related to the fix at hand.

Reply:
I'd go with reverting the changes altogether to make the actual fix/improvement of this PR more apparent, and maybe making the command structure consistent on a separate PR.